content/NavCoin Website/How to add a translation.en.md

```diff
@@ -0,0 +1,173 @@
+
+Some more information on forking repositories can be found in the GitHub help documentation.
+
+https://help.github.com/en/articles/fork-a-repo
+
+### Clone the repository and create a branch
+
+You can use either the Git command line or a GUI like [GitKraken](https://www.gitkraken.com) to clone your forked repository to you local computer and create a branch to make the changes in.
+
+
+### Duplicate the pages you want to translate
+
+The translateable pages are located in the `/content` directory. In this example we will translate the page "Buy NavCoin".
+
+Create a duplicate of the page you want to translate and rename it to have the ISO 639-1 code as part of the file name. For example, if we want to make a translation for Portugeuse we would copy the English file located here;
+
+`/content/buy-navcoin/index.en.md`
+
+And replace "en" with "pt" to have the new file;
+
+`/content/buy-navcoin/index.pt.md`
+
+Do not delete or overwrite the index.en.md file, both need to exist side by side in the project folder.
+
+```
+/content/buy-navcoin/index.en.md
+/content/buy-navcoin/index.pt.md
+```
+
+### Translate the header section
+
+Begin translating the page into your chosen language. Most files have a header section with variables which looks something like this;
+
+```
+---
+title: "Buy NavCoin"
+date: 2018-03-20T08:09:08+13:00
+draft: false
+type: "default"
+author: ""
+description: "NavCoin is publicly mined and ran no token sale. The only way to buy NavCoin is to purchase it from a third party or exchange."
+---
+```
+
+It is important to not change what is to the left of the ":" symbol as these are the variable names used in the code. Usually you only need to translate the text in quotes for `title` and `description`. For example, the Spanish translation of this header section would look like this;
+
+```
+---
+title: "Comprar Navs"
+date: 2018-03-20T08:09:08+13:00
+draft: false
+type: "default"
+author: ""
+description: "NavCoin es un criptoactivo generado públicamente y no ha sido pre-minado/vendido. La única manera de comprar NavCoin es a través de una casa de intercambio tercerizada."
+---
+```
+
+Some pages have different variables in the header section, some of which should not be translated. If you're unsure, ask on discord. Some of the properties which shouldn't be translated in various template headers include;
+
+```
+draft: {true|false}
+type: "default"
+status: {upcoming|in-progress|testing|completed}
+layout: {roadmap|null}
+```
+
+### Translate the rest of the pages
+
+The site is largely built using templates that are rendered with code blocks that look similar to this;
+
+```
+{{</* merchant
+    titleText="CoinPayments"
+    imgSrc="/images/merchants/mer-coinpayment.png"
+    text="Over 70 merchants accept NAV using the CoinPayments merchant&nbsp;gateway."
+    linkUrl="https://www.coinpayments.net/store-directory-coin-NAV"
+*/>}}
+```
+
```

Comment:
You could add something here about how URLs and File paths shouldn't be translated too